packages/roosterjs-editor-dom/lib/index.ts

```diff
@@ -53,6 +53,7 @@ export { default as adjustInsertPosition } from './utils/adjustInsertPosition';
 export { default as createElement, KnownCreateElementData } from './utils/createElement';
 export { default as moveChildNodes } from './utils/moveChildNodes';
 export { default as setListItemStyle } from './utils/setListItemStyle';
+export { default as getTableFormatInfo } from './utils/getFormatTableInfo';
```

Comment:
Move this function into VTable.ts and do not export it unless we really want people to operate on this property